test/io/test_pcs_converter.py

```diff
@@ -436,5 +494,5 @@ def test_read_write(self):
             fh.write(pcs_string)
         with open(name, 'r') as fh:
             pcs_new = pcs.read(fh)
-        assert (pcs_new, cs)
+        self.assertTrue(pcs_new, cs)
```

Comment:
I agree that the old check did nothing, but I think the new one is broken, too. I think you rather want `self.assertEqual(pcs_new, cs)` as the current one only check that the pcs_new string is not empty.